### PR TITLE
feat: remove ionicons

### PR DESCRIPTION
**Pull request by vitorhfs**
https://app.asana.com/0/1182800557000793/1202769114184314

### CODE REVIEW

**Review by Artfloriani — CHANGES_REQUESTED** [review]
Hey, as mentioned one Slack I believe we should use `<slot>` here. Using it would allow developers to use any icon that they want, even emojis :) 

A suggestion is to add CSS to limit the max-width and max-height of the content inside the slot. You could even add some rules specifically to  and <ion-icon>.

File: readme.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ _JavaScript_
 ```js
 const popoverDropdown = document.querySelector('popover-dropdown');
 popoverDropdown.firstOption = 'Deutsch';
-popoverDropdown.icon = 'globe-outline';
+popoverDropdown.icon = './assets/img/globe-outline.svg';
```
Comment (Artfloriani) [inline readme.md:41]: Is this still relevant?

**Review by Artfloriani — CHANGES_REQUESTED** [review]
Just a minor request from a leftover of the previous changes. Also, I have two questions:

1. Did you test if this solution works well with ion-icons? 
2. Did you test if it will work well in cases where we do not provide any icon on this new `slot`?

File: readme.md (hunk 1 above)
Comment (vitorhfs) [reply readme.md:41]: It isn't, forgot to remove :/

**Comment by vitorhfs** [comment]
> Just a minor request from a leftover of the previous changes. Also, I have two questions:
> 
> 1. Did you test if this solution works well with ion-icons?

Yeah! Works great, need to adjust the font-size as any `ion-icon` implementation

> 2. Did you test if it will work well in cases where we do not provide any icon on this new `slot`?

Empty works great as well.

**Review by Artfloriani — APPROVED** [review]

**Pull request merged**